src/Tribe/Tracker.php

```diff
@@ -282,8 +282,12 @@ public function filter_watch_updated_meta( $check, $post_id, $meta_key, $meta_va
 		// If we got here we will update the Modified Meta
 		$modified[ $meta_key ] = $now;
 
+		// Avoid loops!
+		remove_filter( 'update_post_metadata', [ $this, 'filter_watch_updated_meta' ], PHP_INT_MAX - 1 );
```

Comment:
No need to use `PHP_INT_MAX` give it a normal priority that is not 10, and we are all good.